### ISSUE TITLE
OpenTx 2.2.2 - LS and RS not found in companionTX and mixer inputs missing

### ISSUE THREAD

**Issue by jreis1** [opentx/opentx#6176]
Opentx 2.2.2 stable version. 
companionTX for macOS. 

I'm here to report 2 issues, that may be or not related. 

I've a model set on my taranis X9D+ in which LS and RS side buttons are attributed to 2 PWM channels in the mixer.

1) Everything works great, however when importing the model from Taranis to Companion to edit it easily via the software, the LS and RS inputs disappear from the mixer if those inputs are edited and saved.  They are also not on the inputs list.  If not modified in anyway,  those mixers inputs are OK. 

Please attached pictures. 

Orginal model in Radio:
![img_1563](https://user-images.githubusercontent.com/25162166/45715487-4909e180-bb8c-11e8-9ed0-2f8791e39887.JPG)

Model Downloaded to Companion
<img width="548" alt="screen shot 2018-09-18 at 21 28 33" src="https://user-images.githubusercontent.com/25162166/45715217-98034700-bb8b-11e8-8571-64f5f062b720.png">
<img width="626" alt="screen shot 2018-09-18 at 21 28 54" src="https://user-images.githubusercontent.com/25162166/45715218-98034700-bb8b-11e8-831c-c74e0e7c1c75.png">
<img width="348" alt="screen shot 2018-09-18 at 21 29 05" src="https://user-images.githubusercontent.com/25162166/45715219-989bdd80-bb8b-11e8-9ad6-07c640319205.png">

**Comment by jreis1** [comment]
As consequence of the issue reported above, when re-uploading the edited model from the CompanionTX to the radio, some input lines from the mixer disappear. 

Here the File that I've edited in CompanionTX 
<img width="735" alt="screen shot 2018-09-18 at 21 49 23" src="https://user-images.githubusercontent.com/25162166/45715769-eb29c980-bb8c-11e8-9f59-6eaebc4aac63.png">

Here what can be seen in Taranis Mixer: (line CH9 to CH11 missing)
![img_1564](https://user-images.githubusercontent.com/25162166/45715812-0eed0f80-bb8d-11e8-8c70-122ff0709a5f.JPG)

However, if downloading again the same model from the Radio to OpenTx, the lines that are missing on the mixer in the radio are still visible!

**Comment by kilrah** [comment]
You likely have them disabled in your radio settings.

![screenshot_x9d _18-09-18_23-21-19](https://user-images.githubusercontent.com/6065069/45717226-92acf900-bb99-11e8-9ad2-d3231c637ef0.png)

**Comment by KRCNZ** [comment]
Hello kilrah - I'm not sure this is closed. You may have found the fix but what caused it?
I have just had the same issue as above. Is it related to 2.2.2??

**Comment by kilrah** [comment]
There is no fix becasue there is no problem, he just likely had wrong settings in the menu I posted about above.

**Comment by KRCNZ** [comment]
kilrah, maybe he did, so please advise why my settings have mysteriously changed since I flashed my system with 2.2.2? I have been running RS and LS on my X9D+ for 3 years - and all of a sudden my hardware profile has changed - the only difference is I have flashed it with 2.2.2?

**Comment by 3djc** [comment]
You are not saying which version you are coming from, but earlier version did not care much about those settings, now it does matter and needs to be right

**Comment by KRCNZ** [comment]
3djc - OK that is a thought - thank you. I came from 2.2.1 (opentx-x9d+-lua-luac-en-2.2.1). The version I am running now is 2.2.2 (a5de713b). We have only upgraded because we will be playing around with R9M modules and an X-Lite.

Obviously not a wide spread issue otherwise everyone would have the problem. I have just been into work and we have another 3 x X9D+ there and another here - all on the same version. 2 out of the 5 Tx's have the symptoms - so that makes it a bit trickier...

The reason I have voiced an opinion on this thread is I dont think issues should be closed without some explanation giving a reason as to why - or saying "we dont know - we'll leave it open for a little while to see if anyone else has an issue"...
